index.js

```diff
@@ -60,6 +63,59 @@ const getWslDrivesMountPoint = (() => {
 	};
 })();
 
+/**
+Get the PowerShell executable path in WSL environment.
+
+@returns {Promise<string>} The absolute path to the PowerShell executable in WSL.
+*/
+const getPowershellPathFromWsl = async () => {
+	const mountPoint = await getWslDrivesMountPoint();
+	return `${mountPoint}c/Windows/System32/WindowsPowerShell/v1.0/powershell.exe`;
+};
+
+/**
+Get the default browser name in Windows from WSL.
+
+@returns {Promise<string>} Browser name.
+*/
+async function getWindowsDefaultBrowserFromWsl() {
+	const powershellPath = await getPowershellPathFromWsl();
+
+	const rawCommand = `$prog = Get-ItemProperty -Path 'HKCU:\\Software\\Microsoft\\Windows\\Shell\\Associations\\UrlAssociations\\http\\UserChoice' | Select-Object -ExpandProperty ProgId
+	Write-Output $prog
+	`.trim();
```

Comment:
Does it work if we do:

```suggestion
	const rawCommand = `
$prog = Get-ItemProperty -Path 'HKCU:\\Software\\Microsoft\\Windows\\Shell\\Associations\\UrlAssociations\\http\\UserChoice' | Select-Object -ExpandProperty ProgId
Write-Output $prog
		`.trim();
```

That would be the most readable.

Reply:
I cannot test this myself, but AI said it could be simplified to:

```js
const rawCommand = '(Get-ItemProperty -Path "HKCU:\\Software\\Microsoft\\Windows\\Shell\\Associations\\UrlAssociations\\http\\UserChoice").ProgId';
```

Reply:
I tested this command, and it works well!
I’ll go ahead and switch to using this one.